src/Admin/AdminInterface.php

```diff
@@ -102,11 +104,12 @@ public function setTemplates(array $templates): void;
      */
     public function setTemplate(string $name, string $template): void;
 
+    public function setModelManager(ModelManagerInterface $modelManager): void;
```

Comment:
Why is this needed ?

Reply:
Not sure tbh :), I saw `setModelManager` and 'setManagerType` additions in @jordisala1991  PR